src/pages/AdoptablePetsDetails.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PetDetail from "../components/PetDetail";
+import PetSidebar from "../components/PetSidebar";
+import { PetItems } from "../files/PetsData";
+
+const AdoptablePetsDetails = props => {
+  const { animalName = "Bubbles", animalId } = props.match.params;
```

Comment:
so name should come from the filter item below, not the url.

Ideally we would only be passing the id (might have to pass status). Then an ajax call will get the pet details. This will allow us to share these urls.

Reply:
I believe this is done